wrappers/java/src/main/java/org/hyperledger/indy/sdk/metrics/Metrics.java

```diff
@@ -0,0 +1,56 @@
+package org.hyperledger.indy.sdk.metrics;
+
+import com.sun.jna.Callback;
+import org.hyperledger.indy.sdk.IndyException;
+import org.hyperledger.indy.sdk.IndyJava;
+import org.hyperledger.indy.sdk.LibIndy;
+import java.util.concurrent.CompletableFuture;
+
+public class Metrics extends IndyJava.API {
+
+    private Metrics() {
+
+    }
+
+    /*
+     * STATIC CALLBACKS
+     */
+
+    /**
+     * Callback used when method with string result completes
+     */
+    private static Callback stringCompleteCb = new Callback() {
+
+        @SuppressWarnings({"unused", "unchecked"})
```

Comment:
Why do we need these?

Reply:
This annotation is used for all callbacks in libindy. I suggest leaving it.